railties/test/commands/credentials_test.rb

```diff
@@ -122,6 +128,12 @@ class Rails::Command::CredentialsCommandTest < ActiveSupport::TestCase
     assert_no_match(/secret_key_base/, output)
   end
 
+  test "show command from a non-development environment trying to access another environment credentials" do
+    switch_env("RAILS_ENV", "staging") do
+      output = run_show_command(environment: :production)
+      assert_match(/You are accessing production credentials from the staging environment\.\nDid you mean to run `rails credentials:show --environment staging`\?\nFor more information try `rails credentials:help`/, output)
```

Comment:
(If applying previous suggestion.)

```suggestion
      assert_match(/You are accessing production credentials from the staging environment\.\nDid you mean to run `bin/rails credentials:show --environment staging`\?\nFor more information try `bin/rails credentials:help`/, output)
```